### PR TITLE
Update configurations for API Manager 3.0.0 all-in-one

### LINKED ISSUE #86: Create resources for API Manager 3.0.0
**Description:**
Ansible resources should be created for API Manager 3.0.0

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+store_api_password = "{{ stream_processor_rest_api_password }}"
+
+[apim.key_manager]
+service_url = "{{ api_store_server_url }}"
+
+[apim.oauth_config]
+revoke_endpoint = "{{ oauth_configs_revoke_api_url }}"
+
+[apim.devportal]
+url = "{{ api_store_url }}"
```
Comment (ctienshi) [inline roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2]: Can we use the parameter as devportal URL as store is no longer used

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+[apim.throttling]
+receiver_url = "{{ traffic_manager_receiver_url }}"
+receiver_auth_url = "{{ traffic_manager_auth_url }}"
+service_url = "{{ api_store_server_url }}"
```
Comment (ctienshi) [inline roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2]: devportal url

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+enable = "{{ analytics_enabled }}"
+receiver_urls = "{{ stream_processor_url }}"
+receiver_username = "{{ stream_processor_username }}"
+receiver_password = "{{ stream_processor_password }}"
+store_api_url = "{{ stream_processor_rest_api_url }}"
+store_api_username = "{{ stream_processor_rest_api_username }}"
+store_api_password = "{{ stream_processor_rest_api_password }}"
+
+[apim.key_manager]
+service_url = "{{ api_store_server_url }}"
```
Comment (ctienshi) [inline roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2]: devportal url

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 (hunk 2 above)
Comment (Savidude) [reply roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2]: Fixed https://github.com/wso2/ansible-apim/pull/87/commits/0c687c80ac60c4e50d2658064c7a171b09366e21

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 (hunk 3 above)
Comment (Savidude) [reply roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2]: Fixed https://github.com/wso2/ansible-apim/pull/87/commits/0c687c80ac60c4e50d2658064c7a171b09366e21

**Pull request merged**